### DISCUSSION TITLE
Could a "Weapon Handling/Movement Style" be possible?

### DISCUSSION THREAD

**Discussion by Garry4713** [xesdoog/YimActions#30] [Ideas]
For example, the way Michael and our Online player hold shotguns and rifles in combat is nearly identical (and generic) while Trevor and Franklin kinda have their guns pointed up. Let's say I wanted the Online player to have Lester's casual walking animations, but have Trevor's combat stance during gunfights instead, that'd be really cool. Also I think there should be more variety for the Movement Styles option we already have, considering we already have move_heist_lester for Lester's anims I think we should get move_p_m_zero, move_p_m_one , move_p_m_two for Michael, Franklin and Trevor respectively (could be in the wrong order but hey). Thanks for the script I had way more fun than I was supposed to lol

**Comment by xesdoog** [comment]
I'm glad you've enjoyed this fun little experiment! 

Everything you've mentioned is possible but it would require some time to implement which I currently don't have due to life being a constant pain in my ass. I will keep this in mind though!

**Reply by Garry4713** [reply]
Hey keep doin' what you doin' and take breaks when you need them bro, y'all modding people the reason I still play this hot baked microwaved dumpster of a game lol. Still can't understand why the mf rockstar can't add simple features/qol additions that makes the game so much more enjoyable wtf. Good luck brody
